doc/classes/Control.xml

```diff
@@ -117,6 +117,16 @@
 			<argument index="1" name="data" type="Variant">
 			</argument>
 			<description>
+				Godot will call this method on your behalf to ask if [code]data[/code] from a Control's [method get_drag_data] can be dropped at [code]position[/code]. Position is local to this Control.
+				Do not handle the data in this method, only check if able. Return [code]true[/code] if able and [code]false[/code] if unable to handle data. Handle the dropped data in [method drop_data].
```

Comment:
Some redundancy here, and too many repeats of "handle":

This method should only be used to test the data. Process the data in [method drop_data].